examples/compare_simulated_and_physical_robot/main_simulation.py

```diff
@@ -0,0 +1,34 @@
+"""Simulate the modular robot for comparison with the physical counterpart."""
+from revolve2.ci_group import terrains
+from revolve2.ci_group.modular_robots_v2 import gecko_v2
+from revolve2.ci_group.simulation import make_standard_batch_parameters
+from revolve2.experimentation.rng import make_rng_time_seed
+from revolve2.modular_robot import ModularRobot
+from revolve2.modular_robot.brain.cpg import BrainCpgNetworkNeighborRandom
+from revolve2.modular_robot_simulation import ModularRobotScene, simulate_scenes
+from revolve2.simulators.mujoco_simulator import LocalSimulator
+
+
+def main() -> None:
+    """Run the checking."""
+    rng = make_rng_time_seed()  # Make a rng generator for the brain.
+
+    """Here we initialize the Modular Robot."""
+    body = gecko_v2()
```

Comment:
so seeing this is purely to test the robot, can we maybe make this whole thing a standard function? 'test_robot_in_simulation' or something? you provide a simulator and the robot body and that's it. the brain is irrelevant, as are most(?) of the batch parameters for this specific purpose

Reply:
should this function be within the script or somewhere else?